README.md

```diff
@@ -31,34 +31,62 @@ Note:
 Version 3 uses the cordova npm plugin [phonegap-plugin-push](https://github.com/phonegap/phonegap-plugin-push#pushnotificationinitoptions)
 
 ## Config
-Add a `config.push.json` file in your project and configure credentials / keys / certificates:
+Use the `Push.Configure` function on client and server.
 
+On the client
 ```js
-{
-  "apn": {
-    "passphrase": "xxxxxxxxx",  
-    "key": "apnProdKey.pem",
-    "cert": "apnProdCert.pem"
+Push.Configure({
+  android: {
+    senderID: 12341234,
+    alert: true,
+    badge: true,
+    sound: true,
+    vibrate: true,
+    clearNotifications: true
+    // icon: '',
+    // iconColor: ''
   },
-  "apn-dev": {
-    "passphrase": "xxxxxxxxx",
-    "key": "apnDevKey.pem",
-    "cert": "apnDevCert.pem"
-  },  
-  "gcm": {
-    "apiKey": "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx",
-    "projectNumber": xxxxxxxxxxxx
+  ios: {
+    alert: true,
+    badge: true,
+    sound: true
+  }
+});
+```
+
+Additionally you have to touch `mobile-config.js`
+```js
+App.configurePlugin('phonegap-plugin-push', {
+  SENDER_ID: 12341234
+});
+```
+*This is due to changes in the cordova plugin it self*
+
+Server:
+```js
+Push.Configure({
+  apn: {
+    certData: Assets.getText('apnDevCert.pem'),
```

Comment:
I would rename `apnDevCert.pem` to `apnCert.pem`. So deploy tool will be in-charge to to put environment dependent content to `apnCert.pem` file